jarbas/core/tests/shared_tests.py

```diff
@@ -0,0 +1,29 @@
+from unittest.mock import call
+
+from django.test import TestCase
+
+from jarbas.core.models import Reimbursement
+
+# Serializer methods: TestSerializer
+
+
+def test_serializer(TestCase, obj, expected, input):
+    serialized = obj.serialize(input)
+    TestCase.assertEqual(serialized, expected)
```

Comment:
Instead of passing `TestCase` I'd go for a Jarba's `TestCase` every test on Jarbas inherit from:

```python
from django.test import TestCase as DjangoTestCase


class TestCase(DjangoTestCase):

    def test_serializer(self, obj, expected, input):
        serialized = obj.serialize(input)
        self.assertEqual(serialized, expected)
```